fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -384,6 +386,9 @@
     @VariableMgr.VarAttr(name = ENABLE_LATERAL_VIEW, needForward = true)
     public boolean enableLateralView = false;
 
+    @VariableMgr.VarAttr(name = EXEC_SCHEDULING_STRATEGY)
+    public String execSchedulingStrategy = ExecSchedulingStrategy.EVENLY_BACKEND.name();
```

Comment:
For `String` type session variable, user may set any unexpected values.
I think you refer `sql_mode` or `runtime_filter_type`.
And also refer to the `VariableVarConverters.java`

Reply:
Resolved.